pepper-apis/src/test/java/org/broadinstitute/ddp/db/housekeeping/dao/KitCheckDaoTest.java

```diff
@@ -49,6 +49,9 @@ public static void setUp() {
             testData = TestDataSetupUtil.generateBasicUserTestData(handle);
             userGuid = testData.getUserGuid();
             studyGuid = testData.getStudyGuid();
+            JdbiMailAddress jdbiAddress = handle.attach(JdbiMailAddress.class);
+            jdbiAddress.findDefaultAddressForParticipant(userGuid)
+                    .ifPresent(address -> jdbiAddress.deleteAddress(address.getId()));
         });
```

Comment:
The test testCheckForKits_userEnrolled_missingMailAddress started failing consistently. I believe that the test was expecting that the user it was using did not have a mailing address. But because now the split test functionality changes the order of tests, it is likely that another test added an address. This change ensures test sets things up as expected.